src/popup/components/Modals/ConfirmRawSign.vue

```diff
@@ -1,5 +1,6 @@
 <template>
   <Modal
+    show
```

Comment:
How was it working before without `show`?

Reply:
`show` was introduced in `feature/migrate-to-vue3` when we removed the Vue migration build. In Vue3 when `transition` is used as root it wont trigger animations the way we were using it [[see](https://v3-migration.vuejs.org/breaking-changes/transition-as-root.html)]. 
`show` was set to true through `modals.ts/openModal()` but some modals can be accessed directly (without being opened from `openModal()`) so they need the `show` prop otherwise they don't show up